### DISCUSSION TITLE
How to grab both date range values client side quickfilter?

### DISCUSSION THREAD

**Discussion by rchawla-alphabyte** [serenity-is/Serenity#5962] [Help]
Hello,

The issue i'm facing is that I'm unable to retrieve the **end date** range in the getQuickFilters client side method

the variable 'value' is the first date
![image](https://user-images.githubusercontent.com/76563021/132719479-29738686-c883-46c5-a08c-d1671df388d4.png)
![image](https://user-images.githubusercontent.com/76563021/132719646-ab59b351-1476-4fc8-9dc2-e6ec430b76ec.png)

**Comment by edwardch** [comment]
Check out https://serenity.is/demo/BasicSamples/StoredProcedureGrid

The code in the project has been updated since but the code for the `dateRangeQuickFilter` is in this question #4640
